lib/Wikia/src/Tracer/WikiaTracer.php

```diff
@@ -310,7 +314,7 @@ public function getPublicHeaders() {
 			self::LEGACY_TRACE_ID_HEADER_NAME => $this->traceId,
 			// pass the current span ID to the subrequest, it will be logged as parent_span_id there
 			self::PARENT_SPAN_ID_HEADER_NAME => $this->spanId,
-			self::SHIELDS_HEADER_NAME => $this->sjcShields,
+			self::SHIELDS_HEADER_NAME => $this->sjcShieldsHealthStatus,
```

Comment:
sjcShieldsHealthStatus containt 1 if the shields are unhealthy. Isn't it misleading that we flipped the value here, so the "X-SJC-shields-healthy" is set to 1 if the status is unhealthy?